src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -63,7 +60,7 @@ const ArticleTimestamp = ({
     prefix: articleTimestampPrefix,
   };
 
-  const Wrapper = popOut ? PopOutGridItemMedium : GridItemConstrainedMedium;
+  const Wrapper = popOut ? PopOutGridItemMedium : GridItem;
```

Comment:
I was trying to understand why this had changed from `GridItemConstrainedMedium` to `GridItem` and I came across `GridItemMedium`.

What is the difference between `GridItem` and `GridItemMedium`? They appear to be using a very similar configuration.

Reply:
The difference was that `GridItem` applied margins and `GridItemMedium` had no margins. I have removed `GridItemMedium` and added `enableMargins` prop on `GridItem`